examples/evm/simple_functions.py

```diff
@@ -0,0 +1,48 @@
+from seth import *
+################ Script #######################
+
+seth = ManticoreEVM()
+seth.verbosity(0)
+#And now make the contract account to analyze
+# cat  | solc --bin 
+source_code = '''
+pragma solidity ^0.4.13;
+
+contract Test {
+    event Log(string);
+    mapping(address => uint) private balances;
+
+    function Test() {}
+    function target1() public {} 
+    function target2() internal {} 
+    function target3() private {} 
+    function() {}
+
+}
+'''
+#Initialize accounts
+user_account = seth.create_account(balance=1000)
+contract_account = seth.solidity_create_contract(source_code, owner=user_account)
+
+
+symbolic_data = seth.SByte(4) 
+symbolic_value = None 
+seth.transaction(  caller=user_account,
```

Comment:
can we replace this with the new python method call api? also what does this script do? can we add a comment at the top?

Reply:
I wanted to show that we can handle generic transactions. That's an evm transaction.
With 4 bytes of symbolic data (input)
And symbolic value